Hazel/src/Hazel/Events/Event.h

```diff
@@ -29,7 +29,7 @@ namespace Hazel {
 		EventCategoryMouseButton    = BIT(4)
 	};
 
-#define EVENT_CLASS_TYPE(type) static EventType GetStaticType() { return EventType::##type; }\
+#define EVENT_CLASS_TYPE(type) static EventType GetStaticType() { return EventType::type; }\
```

Comment:
Was this intentional? I don't see how this would compile on any system. The point of `##type` is to concatenate whatever type you pass to `EVENT_CLASS_TYPE`

Reply:
You don't need token pasting for it, your not merging things its easily identifiable by the preprocessor